### ISSUE TITLE
God, I miss Power Toggles... is it dead?

### ISSUE THREAD

**Issue by hoderer** [sunnygoyal/PowerToggles#19]
I've been using POWER TOGGLES for ages and I'm very sad that development has stopped. I installed a Android 7.0 rom for a friend (huawei p10 lite) and old 6.0.4 version is  no longer working. Pressing on the power toggles buttons has no effect.

**Comment by tobiascollins** [comment]
Yeah that stinks. Such a cool app. Did you try any other older or newer versions?

**Comment by tobiascollins** [comment]
I guess the developer isn't providing support anymore.

**Comment by hoderer** [comment]
I couldnt find any newer version as 6.0.4. do you know any? I highly doubt that there's a newer one, but would be great of course. Didnt find & tried older ones, guess that makes no sense at all for android 7.

**Comment by tobiascollins** [comment]
I'll see what version mine is. Probably the same as yours. Can't hurt to try an older version. Might get lucky.

**Comment by tobiascollins** [comment]
So my version is the latest one also but here's a link to previous versions. 

https://power-toggles.en.uptodown.com/android/old

I also found an even older one. 3.something.  let me know if you want to give it a try.

**Comment by hoderer** [comment]
thanks alot, but it's exactly as I thought: Not working. It was worth the try! thanks again!

**Comment by tobiascollins** [comment]
Yup!
